Infrastructure/Migrations/20220419141436_init database.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Infrastructure.Migrations
 {
-    public partial class FirstMigration : Migration
```

Comment:
You added new migrations and now everyone will have to download them, maybe it's worth deleting them? This issue will be discussed